CODEOWNERS

```diff
@@ -269,7 +269,7 @@
 /bundles/org.openhab.binding.tplinksmarthome/ @Hilbrand
 /bundles/org.openhab.binding.tr064/ @J-N-K
 /bundles/org.openhab.binding.tradfri/ @cweitkamp @kaikreuzer
-/bundles/org.openhab.binding.unifi/ @mgbowman
+/bundles/org.openhab.binding.unifi/ @mgbowman @WatcherWhale
```

Comment:
This file is used to automatically assign new pull request to binding maintainers. So if you intend to start reviewing and maintaining this binding you can add your name here (assuming you did at least communicate with the current binding developer about this) otherwise don't put your name here.

Reply:
Alright, I'll remove my name.